STYLEGUIDE.md

```diff
@@ -32,8 +31,9 @@ Assume the reader is a busy non-native English speaker.
 ### Referring to the Product and Features
 
 - Sourcegraph: main product, prefer using this name unless you need to be more precise
-  - Sourcegraph Server: the form of our product that ships as the `sourcegraph/server` Docker image and runs on a single node
-  - Sourcegraph Data Center: the form of our product that runs on Kubernetes
+  - Sourcegraph Core: This build is the free tier of Sourcegraph built separately from Sourcegraph OSS.
+  - Sourcegraph Enterprise Starter: the tier of Sourcegraph that includes some features required for dev-ops/admins to deploy in their corporate environment.
+  - Sourcegraph Enterprise: the tier of Sourcegraph that includes all enterprise features for dev-ops/admins to deploy Sourcegraph at a large scale. Enterprise includes the cluster deployment option formerly called Data Center.
```

Comment:
Remove "for dev-ops/admins" because

1. it is not clear who these personas are, specifically (ie they aren't defined anywhere)
1. I would say "dev tools team members" is more accurate than "dev-ops", but that's a mouthful
1. saying who is not necessary here